skfda/misc/metrics.py

```diff
@@ -298,7 +315,7 @@ def norm_lp(fdatagrid, p=2, p2=2):
     return res
 
 
-def lp_distance(fdata1, fdata2, p=2, *, eval_points=None):
+def lp_distance(fdata1, fdata2, p=2, *, eval_points=None, check=True):
```

Comment:
If `check` is only for internal use, call it `_check`:
```suggestion
def lp_distance(fdata1, fdata2, p=2, *, eval_points=None, _check=True):
```

Reply:
Changed.